packages/react-icons/src/utils/createFluentIcon.tsx

```diff
@@ -0,0 +1,31 @@
+import * as React from "react";
+import { FluentIconsProps } from "./FluentIconsProps.types";
+import wrapIcon from "./wrapIcon";
+
+export type FluentIcon = {
+    (props: FluentIconsProps<React.SVGAttributes<SVGElement>>): JSX.Element;
+    displayName: string | undefined;
+}
+
+export const createFluentIcon = (displayName: string, width: string, paths: string[]): FluentIcon => {
```

Comment:
Function name is similar to `./src/utils/fonts/createFluentFontIcon` helper function.